jscomps/EmbedLiteWebrtcUI.js

```diff
@@ -305,6 +309,38 @@ EmbedLiteWebrtcUI.prototype = {
           aSubject.callID
         );
         break;
+
+      case "recording-device-events":
+        let windows = MediaManagerService.activeMediaCaptureWindows;
+        let webrtcMediaInfo = { "video": false, "audio": false};
+
+        for (let i = 0; i < windows.length; i++) {
+          let win = windows.queryElementAt(i, Ci.nsIDOMWindow);
+          let hasCamera = {};
+          let hasMicrophone = {};
+          let screenShare = {};
+          let windowShare = {};
+          let browserShare = {};
+          let mediaDevices = {};
+
+          MediaManagerService.mediaCaptureWindowState(
+            win,
+            hasCamera,
+            hasMicrophone,
+            screenShare,
+            windowShare,
+            browserShare,
+            mediaDevices,
+            true /* aIncludeDescendants */);
+          if (hasCamera.value != MediaManagerService.STATE_NOCAPTURE)
+            webrtcMediaInfo["video"] = true;
```

Comment:
This works as well but personally would prefer ```webrtcMediaInfo.video = true;```

Reply:
Fixed, thank you.